specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/certificates.json

```
@@ -513,7 +513,7 @@
         ],
         "operationId": "ImportCertificate",
         "summary": "Imports a certificate into a specified key vault.",
-        "description": "Imports an existing valid certificate, containing a private key, into Azure Key Vault. The certificate to be imported can be in either PFX or PEM format. If the certificate is in PEM format the PEM file must contain the key as well as x509 certificates. This operation requires the certificates/import permission.",
+        "description": "Imports an existing valid certificate, containing a private key, into Azure Key Vault. This operation requires the certificates/import permission. The certificate to be imported can be in either PFX or PEM format. If the certificate is in PEM format the PEM file must contain the key as well as x509 certificates. Key Vault will only accept a key in PKCS#8 format.",
```

The word "format" gets used a couple times and may be confusing since a line above says PFX or PEM format, then we say PKCS#8 format. So perhaps a better clarification:
```suggestion
        "description": "Imports an existing valid certificate, containing a private key, into Azure Key Vault. This operation requires the certificates/import permission. The certificate to be imported can be in either PKCS#12 (PFX) or PEM file encoding format. If the certificate is in PEM format the PEM file must contain the key as well as X509 certificates. Key Vault will only accept a private key using the PKCS#8 standard syntax.",
```

I also mentioned PKCS#12 because 1) that's what KV actually accepts, and 2) PFX is actually Microsoft's format and, while used interchangably, even openssl uses pkcs12 so it's worth mentioning. But maybe flip the parenthetical if you prefer PFX mainly.